### PR TITLE
Fix broken link in README

### PR DESCRIPTION
**Pull request by bertrandda**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -78,7 +78,7 @@ In Eclipse, create a Debug Configuration by selecting the cgeo application (insi
 
 The Test classes can be found in the project test. Test classes should be located in the same package as
 the class under test.
-Every class can be "Run As" (or "Debug As") an [Android JUnit Test](http://developer.android.com/guide/topics/testing/testing_android.html) from Eclipse.
+Every class can be "Run As" (or "Debug As") an [Android JUnit Test](https://developer.android.com/training/testing/fundamentals.html).
```
Comment (UniQP) [inline README.md]: Did you removed the ` from Eclipse` on purpose?

Comment (pstorch) [reply README.md]: I don't mind if Eclipse is removed here. The default description should target Android Studio now.
But the menu entry in Android Studio is not called `Run As`, it's `Run '<class name>'`.

**Comment by Lineflyer** [comment]
@bertrandda Can you address the change request for this PR which @pstorch adressed in his comment?

**Comment by pstorch** [comment]
@bertrandda still interested?

**Comment by pstorch** [comment]
I've merged it locally and fixed the Eclipse/Android Studio instruction.
Thanks for the updated Link.

**Pull request closed without merging**